### PR TITLE
Add an exercise for classification and clustering first used on the

### PR DESCRIPTION
**Pull request by markotoplak**
Diamond workshop in March 2021

### CODE REVIEW

File: chapters/spec-050-clus-class-exercise/exercise.tex [hunk 1]
```diff
@@ -0,0 +1,41 @@
+\chapter{Exercises}
+
+\section{Clustering}
+
+Open the “Liver-cirrhosis” data set (\widget{Datasets} widget). It represents an a sliced liver tissue. It was sliced so thin that some parts of the image have very low intensities; let's call these "background".
```
Comment (borondics) [inline chapters/spec-050-clus-class-exercise/exercise.tex]: There is a typo here.

Also, the "background" regions originate from the fact that there are actually holes in the tissue, not that it is very thinly sliced.

I would say:

Open the “Liver-cirrhosis” data set from the (\widget{Datasets} widget). It represents a sliced liver tissue. As tissues have holes and they also tear during the slicing some parts of the dataset has very low spectral intensities. Let's call these "background regions".

File: chapters/spec-050-clus-class-exercise/exercise.tex [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+\begin{enumerate}
+\item Remove the background from this data set. For background removal you are likely to need some preprocessing. Remove the background in any way you see fit. Also clustering, as seen below, could be useful for background removal. Try to make it work.
```
Comment (borondics) [inline chapters/spec-050-clus-class-exercise/exercise.tex]: change "background" to "background regions"

just so they don't confuse with spectral background.

File: chapters/spec-050-clus-class-exercise/exercise.tex [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+\item Use Random Forests and Logistic regression. Interpret the models.
+
+\item Observe the effects of different preprocessing on classification performance and model interpretation. Also try models built on the PCA scores.
+
+\end{enumerate}
+
+
+\section{Game of bias}
+
+Start with a dataset with annotated classes. Randomize class values to destroy connections between spectra and classes. Next, think of a novel way to introduce bias in the data set so that AUC become significantly higher than 0.5 even with cross-validation within \widget{Test and Score}.
```
Comment (borondics) [inline chapters/spec-050-clus-class-exercise/exercise.tex]: change to "AUC becomes"

**Comment by markotoplak** [comment]
@borondics, thanks!

**Pull request merged**